src/OrchardCore.Modules/OrchardCore.Forms/Views/Widget-Form.cshtml

```diff
@@ -0,0 +1,10 @@
+@using OrchardCore.Mvc.Utilities;
+@*Until placement.json supports filtering on content types and parts, we need to override the Widget shape to prevent the TitlePart from being rendered. *@
+@{
+    var contentTypeClassName = ((string)Model.ContentItem.ContentType).HtmlClassify();
```

Comment:
What type of Model is it? Can't we use a strongly typed syntax instead of using dynamic?

Reply:
The model is of type `ZoneHolding`. This view is a copy of the Widget shape template, using the content type alternate for the Form content type (the Form content type has the Widget stereotype).